.github/workflows/boulder-ci.yml

```diff
@@ -105,6 +105,30 @@ jobs:
       - name: "Run Test: ${{ matrix.tests }}"
         run: ${{ matrix.tests }}
 
+  govulncheck:
+    runs-on: ubuntu-20.04
+    strategy:
+      # When set to true, GitHub cancels all in-progress jobs if any matrix job fails. Default: true
+      fail-fast: false
+      matrix:
+        GO_VERSION:
+          - "1.20.5"
+          - "1.21rc2"
```

Comment:
Looks like the setup-go action uses a different format for go versions than our own `fetch-and-verify-go.sh` script does: https://github.com/actions/setup-go#v3
> `      go-version: '1.18.0-rc.1' # The Go version to download (if necessary) and use.`

Reply:
Ah, good catch. I ripped out the guts of golang/govulncheck-action to get rid of setup-go. Now we can use the existing matrix successfully.